src/common/io-config/src/s3.rs

```diff
@@ -32,8 +32,8 @@ impl Default for S3Config {
             access_key: None,
             max_connections_per_io_thread: 8,
             retry_initial_backoff_ms: 1000,
-            connect_timeout_ms: 10_000,
-            read_timeout_ms: 10_000,
+            connect_timeout_ms: 30_000,
+            read_timeout_ms: 30_000,
```

Comment:
We also need to update the docs with these new values
https://github.com/Eventual-Inc/Daft/blob/fbc026a0ddb91517039b6bf4e0180a64efe7acc9/src/common/io-config/src/python.rs#L19